index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>YouTube video player page</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS File -->
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>You Tube</h1>
+    </header>
+    <div class="container">
+      <div class="main-video-container">
+        <iframe
+          loop
+          controls
+          class="main-video"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/YXhIaWU3kBg"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+          allowfullscreen
+        ></iframe>
+
+        <h3 class="main-vid-title">Ist video</h3>
+      </div>
```

Comment:
Kindly consider using semantic tags like sections instead of too many divs. This will help improve search engine optimization in your page.